BUILD.gn

```diff
@@ -150,23 +164,14 @@ config("dawn_internal") {
     defines += [ "DAWN_ENABLE_BACKEND_VULKAN" ]
   }
 
-  configs = [ ":libdawn_public" ]
+  configs = [ ":libdawn_public",
+              ":dawn_shared_library_public", ]
```

Comment:
I was worried that adding it to every internal target would break compilation on OSX because of the added ldflags, but it compiles fine.

Reply:
Yes, I just  give a try on this and find it works :-)